### ISSUE TITLE
app is missing theCrag look and feel

### ISSUE THREAD

**Issue by killakalle** [theCrag/tca#87]
**What happened?**
At the moment, using the app, I'm missing theCrag look and feel. To me the TopoGuru theme feels quite different from theCrag. Right now I don't have the impression like I'm using a theCrag app, but I feel like using another app that somehow displays theCrag content.

Do you have any intentions aligning look and feel, theme, UI/UX more towards the theCrag website? 

Example url(s) to reproduce the problem:

Ideally also provide screenshots:
![image](https://user-images.githubusercontent.com/30682796/112767302-3ce12380-9016-11eb-8a91-58e61b631b71.png)

The above is a good example. The only thing website and app have in common regarding the style is the color of the difficulty. Everything else is different:

- Numbering vs no numbering
- colored grade background vs black gradebackground
- grades right vs grades left
- without grade context vs with grade context

**What you expected:**

* [ ] website and app feel like they belong together

**Comment by rouletout** [comment]
Agree, we should move some of that closer, especially where it adds value like readability (the grades are hard to read for me like this for example)

**Comment by bkucsera** [comment]
Ok, it will be changed in v151.
Question: at the moment the grade box contains the type of grade scale, as you see on the above attached screenshot. Shall we keep it like this? (eg. Font 7c+)

**Comment by rouletout** [comment]
Sorry for the delay here, I think we should get rid of the context in the grade field unless it is a different context than the main context of the area.

**Comment by bkucsera** [comment]
We have changed the colours as you see.
In case of the grading systems: i think its better to leave like this in the app, because at the moment the user cannot switch the grading system, so maybe its better to show the grade type as well, not just the grade number.
What do you think?

**Comment by killakalle** [comment]
My comment was not just about the display of the grades. It's also about Fonts, layouts, etc. to make the transition between website and app less of contrast. 

btw, grades here still with black background 
![Screenshot_20210414_234304_com topoguru](https://user-images.githubusercontent.com/30682796/114784854-16441c00-9d7c-11eb-93d5-b7d0ade758a5.jpg)

**Comment by killakalle** [comment]
> i think its better to leave like this in the app, because at the moment the user cannot switch the grading system, so maybe its better to show the grade type as well, not just the grade number.
What do you think?

@bkucsera 
Not being able to switch the grade context does not mean that users do not know in which grade context the current crag is. When you are in your regular home crags you know the grade context. When you go abroad and climb in Australia, you know they have a different grade context. Most  common grade contexts can easily be recognized just by their formats:

Example:
6a+
27
5.10b
9-

The website also manages to not show the grade context - because it is not necessary. The website shows the grade context when there are exceptions, e.g. a route is shown in a different grade context than the rest of the crag. And then this is usually an indicator for a wrong grade context of that route.

**Comment by bkucsera** [comment]
We have deleted the grade_type, as you asked. It will be out in v153 soon.

**Comment by rouletout** [comment]
I think this was not done, I ad a few points from my side here that should be done:

- [ ] include number of bolts in route list
- [ ]  include stars in route list
- [ ]  remove grade scale (at least for most common such as YDS as they take lost of space)
- [ ]  maybe add route gear style (eg mixed crgas of trad and sport)
